src/bsp/src/posix/src/TCPUartMock.cpp

```diff
@@ -2,14 +2,17 @@
 #include "BSPUtils.h"
 #include <FreeRTOS.h>
 #include <FreeRTOSConfig.h>
-#include <cstring>
 #include <freertos-utils/BaseTask.h>
 #include <ros/ros.h>
 #include <task.h>
 
 void TCPUartMock_listenTask(void* param) {
     auto* test = static_cast<TCPUartMock*>(param);
     test->waitForClient();
+
+    while (true) {
+        vTaskDelay(1000);
+    }
```

Comment:
why do we keep the task alive?

Reply:
Because the kernel sometimes stops working if we don't hahaha
Will add a comment

Reply:
F